tests/org.eclipse.n4js.bugreports.tests/bugreports-tests/GHOLD_0070_N4Classifier_members_returns_wrong_members.n4js.xt

```diff
@@ -74,18 +74,18 @@ only non-static: fi, mi
 only non-static: fi, mi
 also static    : fi, fiStatic, mi, miStatic
 ====
-only non-static: fc, mc
-only non-static: fc, mc
-also static    : fc, fcStatic, mc, mcStatic
+only non-static: mc, fc
+only non-static: mc, fc
+also static    : mc, fc, mcStatic, fcStatic
 ====
 without consumed/inherited: 
 without consumed/inherited: 
 with consumed             : mi, fi
-with inherited            : fc, mc
-with consumed & inherited : mi, fi, fc, mc
+with inherited            : mc, fc
+with consumed & inherited : mi, fi, mc, fc
 ====
-with consumed & inherited, only non-static: mi, fi, fc, mc
-with consumed & inherited, also static    : mi, fi, fc, fcStatic, mc, mcStatic
+with consumed & inherited, only non-static: mi, fi, mc, fc
+with consumed & inherited, also static    : mi, fi, mc, fc, mcStatic, fcStatic
```

Comment:
What about the order here? Members that are "detected" on the prototype/constructor do no longer have a defined order, right? Does that mean these tests could break depending on node version or even depending on some internal non-deterministic behavior of the JS engine?

(depends on `Object#getOwnPropertyNames()`)